package/entrypoint.sh

```diff
@@ -6,4 +6,7 @@ if [ ! -e /run/secrets/kubernetes.io/serviceaccount ] && [ ! -e /dev/kmsg ]; the
     exit 1
 fi
 rm -f /var/lib/rancher/k3s/server/cred/node-passwd
+if [ -x "$(command -v c_rehash)" ]; then
```

Comment:
what if the command `c_rehash` exists but is not executable? Currently, we just ignore it.

Reply:
The binary is included when we install `openssl` in the image, so I think it will be executable if it is there.

Reply:
Ok, so we do not need to concern about it.